pennylane/wires.py

```diff
@@ -64,7 +51,7 @@ def _process(wires):
     )
 
 
-class Wires(Sequence):
+class Wires(OrderedSet):
     r"""
     A bookkeeping class for wires, which are ordered collections of unique objects. The :math:`i\mathrm{th}` object
     addresses the :math:`i\mathrm{th}` quantum subsystem.
```

Comment:
I'm assuming this docstring needs an update after all these changes?

Reply:
I slightly changed it now, should there be more information on the class?

Reply:
Looks good to me :+1: